File: 151002/Samusenko/Lab_1/src/main/java/com/example/dc_project/model/entity/Issue.java
package com.example.dc_project.model.entity;

import com.example.dc_project.model.AEntity;
import lombok.Getter;
import lombok.Setter;
import lombok.experimental.SuperBuilder;
import java.time.LocalDateTime;

@Getter
@Setter
@SuperBuilder

public class Issue extends AEntity {
    private Long userId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
}

File: 151003/Barylko/REST/TaskREST/Models/Issue.cs
using System.ComponentModel.DataAnnotations;

namespace TaskREST.Models;

public class Issue
{
    public long Id { get; set; }
    public long UserId { get; set; }
    [MaxLength(32)] public string Title { get; set; } = string.Empty;
    [MaxLength(2048)] public string Content { get; set; } = string.Empty;
    public DateTime Created { get; init; }
    public DateTime Modified { get; set; }
}